resources/assets/uad_lists.json

```
@@ -24827,8 +24827,8 @@
     "labels": []
   },
   "com.vivo.vivo3rdalgoservice": {
-    "description": "ImageAlgoService\nHDR and High contrast in the camera will probably not work based on XDA users",
-    "removal": "Expert",
+    "description": "ImageAlgoService\nHDR and High contrast in the camera?\nProbably safe to remove, so it's debugging app?",
+    "removal": "Recommended",
```

```suggestion
    "removal": "Advanced",
```
If it's for HDR and High Contrast in the camera app, this should not be `Recommended`.